.github/CODEOWNERS

```diff
@@ -3,7 +3,9 @@
 
 # Documentation.
 /docs/ @inertial-frame @klizhentas
+/docs/pages/application-access/ @inertial-frame @klizhentas @r0mant
+/docs/pages/database-access/ @inertial-frame @klizhentas @r0mant
```

Comment:
Didn't know if `/docs/` rule above would match these as well so added all 3 of us here just in case.